### PR TITLE
Connected all Admin widgets

### PR DESCRIPTION
**Pull request by IlyaM1**
1)
1. Connected Authorization with Admin_view_tests_and_users (made TODO: вызывать некст окно с передачей user_arr и test_arr)
2. Connected Admin_view_tests_and_users with Admin_test_view and Admin_user_view

2) Now Clickable_label_with_delete_buttons contains dictionary, NOT id
3) Fixed self.EMPTY_QUESTION bug(this not changes now after generating new question)
4) Changed all paths to css files with absolute path
5) create_new_test/user now creates empty test/user(should also add new test/user in db)
6) Fixed signing as admin

Partly made issue #70

### CODE REVIEW

**Comment by IlyaM1** [comment]
@akrisfx

File: main.py [hunk 1]
```diff
@@ -12,15 +12,16 @@ def main():
 
 if __name__ == '__main__':
     main()
-    cfg = Config()
-    s = SQLInteract(table_name='testcase', filename_db=cfg.config["path"] + '/db/users.db')
-    sign = Signing({'id': 0, 'name': 'ILYA3224', 'password': '1234', 'post': 'pop', 'test': '[]'}, s)
-    print(cfg.config)
+    # cfg = Config()
+    # s = SQLInteract(table_name='employees', filename_db=cfg.config["path"] + '/db/users.db')
+    # sign = Signing({'id': 0, 'name': 'ILYA3224', 'password': '1234', 'post': 'pop', 'test': '[]'}, s)
```
Comment (akrisfx) [inline main.py:17]: за шо

**Review by akrisfx — APPROVED** [review]
норм

File: main.py (hunk 1 above)
Comment (IlyaM1) [reply main.py:17]: а зач?

**Pull request merged**